tower-test/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+//! Mock `Service` that can be used in tests.
+
+extern crate futures;
+extern crate tokio_sync;
+extern crate tower_service;
+
+mod macros;
```

Comment:
does this support getting the macro via an import?

Reply:
Not sure... I would guess so? It would be worth a test. We also probably should update everything to 2018...

Reply:
If it is not, we can update it with #238.